### PR TITLE
docs(portal): review enterprise csmportal

### PR DESCRIPTION
**Pull request by christinaausley**
Technical review of enterprise CSM portal doc.

### CODE REVIEW

**Comment by akeller** [comment]
I'm not sure why the two files in the `/bin/test` folder got added. Can you remove them from the commit? (To be clear, I don't mean delete the file, they might be necessary elsewhere, just not for this PR.)

File: enterprise/content/enterprise-customer-portal.md [hunk 1]
```diff
@@ -7,121 +7,127 @@ menu:
...
 ---
 
-This guide is intended to help you navigate our new Camunda Enterprise Customer Portal. This document includes information about how to get access, how to download the license keys, create temporary license keys and manage support contacts for our support JIRA.
+This guide helps you navigate our Camunda Enterprise Customer Portal. This document includes information about how to gain access, download the license keys, create temporary license keys, and manage support contacts for our support JIRA.
 
-Please note that support will continue to be provided via our [JIRA support system](https://jira.camunda.com/projects/SUPPORT/queues).
+Note that support will continue to be provided via our [JIRA support system](https://jira.camunda.com/projects/SUPPORT/queues).
 
-**Disclaimer**: the Camunda Enterprise Portal is currently in closed beta. If you would like to be part of the beta users, please contact your Customer Success Manager or email customer-success@camunda.com.
+**Disclaimer**: The Camunda Enterprise Portal is currently in closed beta. If you would like to join the beta users, contact your Customer Success Manager or email customer-success@camunda.com.
```
Comment (akeller) [inline enterprise/content/enterprise-customer-portal.md:14]:
```suggestion
**Disclaimer**: The Camunda Enterprise Portal is currently in closed beta. If you would like to join the beta users, contact your Customer Success Manager or email [customer-success@camunda.com](mailto:customer-success@camunda.com).
```

**Comment by christinaausley** [comment]
@lwille Thought on why the two files in the `/bin/test` folder were "added" in this PR?

**Comment by akeller** [comment]
@christinaausley I deleted the swp file with a commit because I couldn't figure out where it was added, but I assume it had something to do with the merge conflict that was still hanging around. 

I screen recorded my interactive rebase so you can see what happened there. When you enter, you are already in the mode where the editor wants to accept commands so hit "i" (for insert mode). This allows you to type. From there drop the commits you want to drop then when you are done hit Esc : wq to write and quit the editor.

For situations like this, I always clone a fresh copy of the entire project and navigate to the branch. That also removes any other weird situations where I may have screwed something up in my local git history that I may not be smart enough to recognize of undo!

**Comment by akeller** [comment]
@christinaausley I deleted the swp file with a commit because I couldn't figure out where it was added, but I assume it had something to do with the merge conflict that was still hanging around. 

I screen recorded my interactive rebase so you can see what happened there. When you enter, you are already in the mode where the editor wants to accept commands so hit "i" (for insert mode). This allows you to type. From there drop the commits you want to drop then when you are done hit Esc : wq to write and quit the editor.

For situations like this, I always clone a fresh copy of the entire project and navigate to the branch. That also removes any other weird situations where I may have screwed something up in my local git history that I may not be smart enough to recognize of undo!

**Comment by tmetzke** [comment]
@christinaausley, sorry for the delay but I was out of office for two weeks. Do you still need my input here?

**Comment by christinaausley** [comment]
> @christinaausley, sorry for the delay but I was out of office for two weeks. Do you still need my input here?

@tmetzke We resolved the rebasing issue, but thoughts on why the two files  in `/bin/test` were auto-added in the first commit?

**Comment by tmetzke** [comment]
@christinaausley, no idea, to be honest. I have never seen files appear by anything I did with the repository like running Hugo or creating commits. Keep an eye on it and when it happens again, try to go through the steps you took, so maybe we can figure out what creates these files.

**Pull request merged**